<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminedit_apt</class>
 <widget class="QDialog" name="Adminedit_apt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="ch_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Drop your Choice</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="choice_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 1</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QGroupBox" name="crt_choice">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Correct Choice</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choice-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Choice-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Choice-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Choice-4</string>
      </property>
     </item>
    </widget>
=======
   <widget class="QRadioButton" name="ch1_rb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="ch3_rb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="ch4_rb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="ch2_rb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
>>>>>>> 25f0b628cd13990f9a1f0c82105a79c4880cbd1c
   </widget>
  </widget>
  <widget class="QGroupBox" name="qn_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>621</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Drop your Question</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Queston</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qn_editor">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Qn.no :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qn_number">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submitbtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>570</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DB Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="qn_db_status">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
